_layouts/default.html

```diff
@@ -5,7 +5,7 @@
   <body>
 
     <div>
-      <a href="{{ site.baseurl }}">
+      <a href="{{ site.baseurl }}/">
```

Comment:
Mmh… it also means that'll break whenever a deployment is made at some other URL. Like, when it'll be deployed at `api.gouv.fr` ;)